### PR TITLE
feat: Add dashboard for Apache APISIX

### PR DESCRIPTION
**Pull request by AlinsRan**
### What does this PR do?

Increase Observability to monitor the throughput and latency of the API.

image

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by drichards-87 — APPROVED** [review]

**Comment by yzhan289** [comment]
Pinging @bisakhmondal to review

File: apache-apisix/assets/dashboards/apache-apisix_overview.json [hunk 1]
```diff
@@ -0,0 +1,480 @@
+{
+    "title": "Apache APISIX Overview",
+    "description": "## Apache APISIX Overview\nApache APISIX integration with Datadog lets you dive deeper into your monitoring runs, helping you stay on top of your API's throughput, latency and more.",
+    "widgets": [
+        {
+            "id": 3577777298296914,
+            "definition": {
+                "type": "image",
+                "url": "/static/images/logos/apache-apisix_small.svg",
```
Comment (bisakhmondal) [inline apache-apisix/assets/dashboards/apache-apisix_overview.json:9]: qq: I might be missing context but I don't see this asset anywhere [?]
Better to give some cdn link, no?

File: apache-apisix/assets/dashboards/apache-apisix_overview.json [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+            "name": "balancer_ip",
+            "prefix": "balancer_ip",
+            "available_values": [],
+            "default": "*"
+        }
+    ],
+    "layout_type": "ordered",
+    "notify_list": [],
+    "reflow_type": "fixed"
+}
```
Comment (bisakhmondal) [inline apache-apisix/assets/dashboards/apache-apisix_overview.json:480]: nit: newline

**Review by bisakhmondal — COMMENTED** [review]
Thanks for doing this, in general lgtm.
Would you mind
1. adding a bug id for the change
2. a screenshot for the change (if it's not too hard, I don't see any good CI for this)

File: apache-apisix/assets/dashboards/apache-apisix_overview.json (hunk 1 above)
Comment (AlinsRan) [reply apache-apisix/assets/dashboards/apache-apisix_overview.json:9]: Sorry, it is displaying properly now, but I can't find the documentation for configuring logos. Maybe I need some help.

image

File: apache-apisix/assets/dashboards/apache-apisix_overview.json (hunk 2 above)
Comment (AlinsRan) [reply apache-apisix/assets/dashboards/apache-apisix_overview.json]: done.

**Comment by AlinsRan** [comment]
> 
> 1. adding a bug id for the change


I'm sorry, I don't quite understand. What is the bug ID you mentioned and how do I add it?

**Review by bisakhmondal — APPROVED** [review]

**Comment by bisakhmondal** [comment]
> What is the bug ID you mentioned and how do I add it?

I meant an issue id (tracking issue for this feature request). Please add it before submitting the change.
Approved

File: apache-apisix/assets/dashboards/apache-apisix_overview.json (hunk 1 above)
Comment (bisakhmondal) [reply apache-apisix/assets/dashboards/apache-apisix_overview.json:9]: > Sorry, it is displaying properly now

Okay, I assume the screenshot is from your local build output?

File: apache-apisix/assets/dashboards/apache-apisix_overview.json (hunk 2 above)
Comment (bisakhmondal) [reply apache-apisix/assets/dashboards/apache-apisix_overview.json]: Thanks

File: apache-apisix/assets/dashboards/apache-apisix_overview.json (hunk 1 above)
Comment (AlinsRan) [reply apache-apisix/assets/dashboards/apache-apisix_overview.json:9]: Datadog is a cloud platform and cannot be built locally.

Comment (AlinsRan) [reply apache-apisix/assets/dashboards/apache-apisix_overview.json:9]: If you need to verify, you can import the JSON file into the Datadog dashboard.

**Comment by leslie-tsang** [comment]
Any update of this PR?

**Comment by bisakhmondal** [comment]
Ship it. It's already approved!

**Comment by EmilyKeer** [comment]
> 

Can the maintainer merge it? Thanks!

**Pull request merged**